test/extended/images/info.go

```diff
@@ -21,6 +22,17 @@ var _ = g.Describe("[sig-imageregistry][Feature:ImageInfo] Image info", func() {
 		}
 	})
 
+	g.BeforeEach(func() {
+		microShift, err := exutil.IsMicroShiftDeployment(oc)
+		o.Expect(err).NotTo(o.HaveOccurred())
+		if microShift {
+			// Test works as oc image info does not require any APIs, but the pod from
+			// where this is executed is a local image to the cluster, which is not
```

Comment:
Any way to pull the image to the cluster?

Reply:
We would need to promote an image with oc inside to a public registry, not sure about that. Also, I wonder if this test should be in sig-cli, there are no checks in this one besides the pod coming up.